inc/file.class.php

```diff
@@ -212,14 +214,13 @@ public function prepareInputForUpdate($input) {
          if ($filename != $this->fields['source']) {
             unlink(FLYVEMDM_FILE_PATH . "/" . $this->fields['source']);
          }
+         // File updated, then increment its version
+         $input['version'] = $this->fields['version'] + 1;
```

Comment:
nice bug where the version was not udated . . post increment instead of pre increment 👎